enterprise_access/apps/bffs/serializers.py

```diff
@@ -152,12 +152,26 @@ class EnterpriseCustomerSerializer(BaseBffSerializer):
     show_integration_warning = serializers.BooleanField()
 
 
+class EnterpriseCustomerUserSerializer(BaseBffSerializer):
+    """
+    Serializer for all linked enterprise customer users
+    """
+    id = serializers.IntegerField()
+    user_id = serializers.IntegerField()
```

Comment:
[very optional nit] could probably nix `user_id` from the response, as I imagine its value will be the same across all `EnterpriseCustomerUsers` in the list and if the frontend needed a user id for the authenticated user, it'd likely get it from `useContext(AppContext).authenticatedUser.userId` vs. this API response.